### ISSUE TITLE
HTTP Fehler 500 "internal error" beim Editieren im Wiki

### ISSUE THREAD

**Issue by cstrotm** [GeraldWodni/forth-ev.de#4]
Wird das Wiki ueber die Navigationsleiste auf der Hauptseite aufgerufen, so schlagen Modifikationen an den Wiki Seiten mit einem Fehler 500 "internal Server error" fehl. Der Editierte Text wurde dann schon uebernommen, der Fehler passiert moeglicherweise bei einem Redirect nachdem die Aenderungen gespeichert wurden.

![Screen Shot 2019-05-15 at 14 11 19 pm](https://user-images.githubusercontent.com/141758/57774764-afdf9a80-771b-11e9-95bd-2bb6fcf1930d.png)

**Comment by GeraldWodni** [comment]
Ja, das kann man reparieren aber Bernd meinte das ist doch gleichzeitig eine Art Schutz ;)

Ich weiß nicht mehr wie die Diskussion über den WikiProxy ausgegangen ist. Ich könnte ihn fixen (das meiste geht ja schon), und es hat den Vorteil, dass der Style besser in die Website passt, man Links rewriten kann und die Navigation viel besser funktioniert als in einem iframe.
Natürlich kann man diese Dinge auch begrenzt mit Javascript fixen (vor allem die Navigation) und den iframe einbinden.

Im Moment hängt das ganze in der Luft weil ich nicht in die falsche Variante Zeit investieren will. Vielleicht kannst du dich mit den anderen Editoren (vor allem Michael) beim nächsten Treffen beraten und dann zurückschreiben:
1. Wir machen den proxy fertig und löschen dabei alle Interaktionsmöglichkeiten zum Anheben der scheinbaren Sicherheit. Man könnte hier auch auf das editieren unter der alternativen URL hinweisen.
2. Wir machen den proxy fertig und bauen reparieren diese Interaktionsmöglichkeiten
3. Wir machen einen iframe

lgg

**Comment by cstrotm** [comment]
Ich verstehe nicht wie das ein Schutz sein kann. Die Aenderungen gehen ja trotzdem durch, d.h. Bots und Trollen koennen das Wiki verunstalten, nur echte Benutzer bekommen eine Fehlermeldung.

Benutzer sollten Aenderungen im Wiki vornehmen koennen.

Ich bin fuer Option 4: das Menue korrekt ohne iframe in DokuWiki einbinden (via JavaScript). Dafuer muss warscheinlich nur eine Zeile im DokuWiki-Template geaendert werden, auch bei Updates haelt sich der Aufwand in Grenzen.

Eine Beratung innerhalb der Regional-Gruppe Essen bringt uns nicht weiter, da die Benutzer die technischen Hintergruende nicht abschaetzen koennen.

Ich werde auf dem neuen Server mal einen Prototyp bauen.

Beste Gruesse

Carsten
